docs/conf.py

```diff
@@ -21,7 +21,7 @@
 import os
 import sys
 cur_dir = os.path.abspath(os.path.dirname(__file__))
-sys.path.insert(0, os.path.join(cur_dir, '..', 'cartoframes'))
+sys.path.insert(0, os.path.join(cur_dir, '..'))
 from cartoframes.__version__ import __version__ as cf_version  # noqa
```

Comment:
I get an `ImportError: cannot find module IPython` when I try to build the docs with just Sphinx and Sphinx-napoleon installed.

So it seems like another problem of importing the module here is that it's dependencies will be looked for. So this line tries to import the version in `__version__` but that imports everything in `cartoframes/__init__.py`. Is there a better way to get the version number without having to import cartoframes?

The version number is set in that file so that it's the source of truth (setup.py pulls it for example and I wanted it to be callable from a script too).